app/presenters/speedy_af/proxy/master_file.rb

```diff
@@ -67,6 +67,7 @@ def supplemental_files(tag: '*')
   end
 
   def captions
+    return nil unless has_captions?
```

Comment:
`has_captions?` is true if the section has legacy captions OR supplemental file captions.  This guard clause will avoid solr calls when the section does not have any captions at all, but if the section has supplemental file captions only it will still attempt to find legacy captions in solr.  This is a compromise to avoid calling fedora but still avoid some solr calls.  Also, I don't believe this method is actually called anywhere.